src/main/java/jenkins/plugins/mattermost/StandardMattermostService.java

```diff
@@ -83,6 +87,7 @@ public boolean publish(String message, String color) {
 				attachment.put("mrkdwn_in", mrkdwn);
 				JSONArray attachments = new JSONArray();
 				attachments.put(attachment);
+				json.put("text", text);
```

Comment:
fixed the indentation here

Reply:
There were some more lines in the same file, and in `src/main/java/jenkins/plugins/mattermost/MattermostService.java`:

* https://github.com/jovandeginste/jenkins-mattermost-plugin/pull/24/files/64d101547b95057f8c9f9a698865795db312a6cc#diff-d650a89249a9787a7fb00ef5125010b1R40
* https://github.com/jovandeginste/jenkins-mattermost-plugin/pull/24/files/64d101547b95057f8c9f9a698865795db312a6cc#diff-6b9ebfa2a5496a99040fedf030b1fc76R6